articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -0,0 +1,218 @@
+
+The thumbprint of the certificate(s) is also required. A guide on how to find the thumbprint of the certificate(s) can be found [here](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/how-to-retrieve-the-thumbprint-of-a-certificate).
+
+## Create a Service Fabric cluster
+
+1. In the favourites panel, select **Create a resource**.
+
+   ![New option in favourites panel](images/azsp_newmenu.png)
+
+2. In the search bar, search for **Service Fabric Cluster**.
+
+   ![Compute option in New blade](images/azsp_computeblade.png)
+
+3. Select **Service Fabric Cluster** and click **Create**.
+
+4. In the **Basics** blade, enter the following information and click **OK**:
+
+   - **Service Fabric Cluster Name** - The name of your cluster.
+
+   - **Node Type Prefix** - Prefix for the name of each Node(VM).
+
+   - **Primary Node Type Size** - Size of the VM scale set for the primary node type.
+
+   - **Additional Node Type Sizes** - For every additional node type add an array with count of nodes in each node type. For example, if you want 2 additional node types with 3 nodes in each, enter each amount separated by comma： 3,3.
+
+   - **Admin Username** - The admin username for each node.
+
+   - **Password** - The admin password for each node.
+
+   - **DNS Service** - DNS service is an optional system service that enables you to map DNS names to a service name and discover other services using the DNS protocol. If you have applications with existing URLs that you intend to use with Service Fabric, DNS service will help you use the standard DNS protocol (as an alternative to the Service Fabric Naming service) for resolving service names.
+
+   - **Repair Manager** - Repair manager allows you to enable patch orchestration on node types with a durability of bronze, which helps keep your virtual machines up to date.
+
+   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For off-line scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group. Note that to create a Service Fabric cluster, the resource group must be empty.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   ![Create Service Fabric cluster > Basics](images/azs-browser-create-sf-basics.png)
+
+5. In the **Network Settings** blade, enter the following information and click **OK**:
+
+   - **Service Fabric TCP Port** - Service Fabric cluster TCP gateway port to be used to connect using Service Fabric client.
+
+   - **Service Fabric HTTP Port** - Service Fabric cluster HTTP gateway port to be used to connect using Service Fabric Explorer.

```

Comment:
Service Fabric cluster HTTP gateway port to use to connect using the Service Fabric Explorer.